nightfall-deployer/contracts/Challenges.sol

```diff
@@ -274,7 +274,7 @@ contract Challenges is Stateful, Key_Registry, Config {
     uint256 lastBlock = state.getNumberOfL2Blocks() - 1;
     for (uint256 i = lastBlock; i >= blockNumberL2; i--) {
       state.popBlockData();
-    }
+    }  
```

Comment:
nit: we don't have a linter running on `.sol` files but this white space should be removed